### ISSUE TITLE
[v4] Block body arrows can not be immediately invoked without a group

### ISSUE THREAD

**Issue by SephReed** [fuse-box/fuse-box#1732]
I'm still figuring out what this error means, but figured it should be findable with a search engine.

```
fuse-box-css:1.0.0{ SyntaxError: [37:1]: Block body arrows can not be immediately invoked without a group
    at report (/Users/seph/Code/ChooseHealth/instride-storefront/node_modules/meriyah/dist/meriyah.umd.js:188:13)
    at parseArrowFunctionExpression (/Users/seph/Code/ChooseHealth/instride-storefront/node_modules/meriyah/dist/meriyah.umd.js:5511:19)
    at parseParenthesizedArrow (/Users/seph/Code/ChooseHealth/instride-storefront/node_modules/meriyah/dist/meriyah.umd.js:5484:14)
    at parseParenthesizedExpression (/Users/seph/Code/ChooseHealth/instride-storefront/node_modules/meriyah/dist/meriyah.umd.js:5310:18)
    at parsePrimaryExpression (/Users/seph/Code/ChooseHealth/instride-storefront/node_modules/meriyah/dist/meriyah.umd.js:4062:22)
    at parseExpression (/Users/seph/Code/ChooseHealth/instride-storefront/node_modules/meriyah/dist/meriyah.umd.js:3522:18)
    at parseAssignmentExpression (/Users/seph/Code/ChooseHealth/instride-storefront/node_modules/meriyah/dist/meriyah.umd.js:3552:25)
    at parseExpressionOrLabelledStatement (/Users/seph/Code/ChooseHealth/instride-storefront/node_modules/meriyah/dist/meriyah.umd.js:2654:14)
    at parseStatement (/Users/seph/Code/ChooseHealth/instride-storefront/node_modules/meriyah/dist/meriyah.umd.js:2633:22)
    at parseStatementListItem (/Users/seph/Code/ChooseHealth/instride-storefront/node_modules/meriyah/dist/meriyah.umd.js:2583:22)
```

Might be this:

```
:root {}
```

**Comment by SephReed** [comment]
It was these lines of code:

```
api.tryAddDevFlag = () => {
	if (api.isDev()) {
		const flag = document.createElement("dev-env");
		flag.innerText = "DEV";
		document.body.appendChild(flag);
	}
}
(document.body && api.tryAddDevFlag())
|| document.addEventListener("DOMContentLoaded", api.tryAddDevFlag);
```

A newline between the two parts was required.  This code does run in browser though.

**Comment by SephReed** [comment]
Actually, this is the solution:

```
api.tryAddDevFlag = () => {
	if (api.isDev()) {
		const flag = document.createElement("dev-env");
		flag.innerText = "DEV";
		document.body.appendChild(flag);
	}
}

{
	(document.body && api.tryAddDevFlag())
	|| document.addEventListener("DOMContentLoaded", api.tryAddDevFlag);
}
```

`meriyah` has a bug.

**Comment by 3cp** [comment]
This is fixed in just released meriyah v1.9.12.

**Comment by charsleysa** [comment]
Closed due to inactivity.
